### PR TITLE
🌐 Add French translation for `deployment/manually.md`

### PR DESCRIPTION
**Pull request by rjNemo**
🤗 

Here is the PR to translate the manual deployment.
See the french translation tracking issue [here](https://github.com/tiangolo/fastapi/issues/1972).

Thanks for the reviews

### CODE REVIEW

File: docs/fr/mkdocs.yml [hunk 1]
```diff
@@ -60,6 +60,8 @@ nav:
 - Tutoriel - Guide utilisateur:
   - tutorial/background-tasks.md
 - async.md
+- Deploiement:
```
Comment (Smlep) [inline docs/fr/mkdocs.yml:63]:
```suggestion
- Déploiement:
```

File: docs/fr/docs/deployment/manually.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+    ```console
+    $ pip install uvicorn[standard]
+
+    ---> 100%
+    ```
+
+    </div>
+
+    !!! tip
```
Comment (Smlep) [inline docs/fr/docs/deployment/manually.md:21]:
```suggestion
    !!! tip "Astuce"
```

File: docs/fr/docs/deployment/manually.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    $ pip install hypercorn
+
+    ---> 100%
+    ```
+
+    </div>
+
+    ...ou tout autre serveur ASGI.
+
+Et exécutez votre application comme vous l'avez fait dans les tutoriels, mais sans l'option `--reload`, par exemple :
```
Comment (Smlep) [inline docs/fr/docs/deployment/manually.md:42]:
```suggestion
Et d'exécuter votre application comme vous l'avez fait dans les tutoriels, mais sans l'option `--reload`, par exemple :
```
> Il vous suffit d'installer...

**Review by Smlep — CHANGES_REQUESTED** [review]

**Review by arthurio — CHANGES_REQUESTED** [review]
Unfortunately it looks like this page has changed quite a lot. https://raw.githubusercontent.com/tiangolo/fastapi/master/docs/en/docs/deployment/manually.md

**Review by axel584 — APPROVED** [review]

File: docs/fr/docs/deployment/manually.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+C'est la base. Mais vous voudrez probablement vous occuper de certaines choses supplémentaires, comme.. :
+
+* la sécurité - HTTPS
+* l'exécution au démarrage
+* le redémarrage
+* la réplication (le nombre de processus en cours d'exécution)
+* la mémoire
+* les étapes précédant le démarrage
+
+Je vous en dirai plus sur chacun de ces concepts, sur la façon d'y réfléchir et sur des exemples concrets avec des stratégies pour les traiter dans les prochains chapitres. 🚀
```
Comment (arthurio) [inline docs/fr/docs/deployment/manually.md:153]:
```suggestion
Je vous en dirai plus sur chacun de ces concepts, sur la façon de les aborder, et donnerai quelques exemples concrets avec des stratégies pour les traiter dans les prochains chapitres. 🚀
```

**Review by arthurio — COMMENTED** [review]
Looks really good overall, I added some nits to thing about and some personal preference suggestions (everything is up for debate 😉 ).
Just a quick question:
Should we keep the titles syntax with upper case main words?
e.g. `## Installer le Programme Serveur` vs. `## Installer le programme serveur`

File: docs/fr/docs/deployment/manually.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
+# Exécuter un serveur manuellement - Uvicorn
+
+La principale chose dont vous avez besoin pour exécuter une application **FastAPI** sur une machine serveur distante est un programme serveur ASGI tel que **Uvicorn**.
+
+Il existe 3 principales alternatives :
+
+* <a href="https://www.uvicorn.org/" class="external-link" target="_blank">Uvicorn</a> : un serveur ASGI haute performance.
+* <a href="https://pgjones.gitlab.io/hypercorn/" class="external-link" target="_blank">Hypercorn</a> : un serveur
+  ASGI compatible avec HTTP/2 et Trio entre autres caractéristiques.
```
[the remark this answers is not in the document]
Comment (rjNemo) [reply docs/fr/docs/deployment/manually.md]: Thanks for the comments all were very useful 👍 

> Should we keep the titles syntax with upper case main words?
> e.g. ## Installer le Programme Serveur vs. ## Installer le programme serveur

In French, we don't use title case so I wouldn't do that.

**Review by arthurio — APPROVED** [review]

File: docs/en/docs/deployment/manually.md [hunk 6]
```diff
@@ -85,7 +85,7 @@ You can then run your application the same way you have done in the tutorials, b
 
     </div>
 
-!!! warning
+!!! Attention
```
Comment (tiangolo) [inline docs/en/docs/deployment/manually.md:88]:
```suggestion
!!! warning
```

The specific keyword is what triggers and shows the correct tooltip. Also, this one is for English.

**Comment by tiangolo** [comment]
Awesome, thank you @rjNemo! :rocket: 

And thanks for the reviews @arthurio, @axel584 :bow: :cake:

File: docs/en/docs/deployment/manually.md (hunk 6 above)
Comment (rjNemo) [reply docs/en/docs/deployment/manually.md]: Thanks for catching this one!

**Pull request merged**